my-index.ejs

```diff
@@ -5,7 +5,7 @@
     <title><%= htmlWebpackPlugin.options.title %></title>
     <script>
         window.devContext = true;
-        window.defaultRoute = "/study";
+        window.defaultRoute = "/newstudy";
```

Comment:
We could still use the same `/study` and change it at servlet (jsp) side

Reply:
We need to add new servlet `newstudy` and `study` route in cbioportal-fronted for the above url to work